src/main/java/org/galatea/starter/Application.java

```diff
@@ -39,5 +44,8 @@ public void run(final ApplicationArguments args) {
     if (!args.containsOption("server.port") && System.getProperty("server.port") == null) {
       throw new MissingOptionException("Server port must be set via command line parameter");
     }
+    log.info("Started successfully, you can view swagger UI here: {}:{}/",
+        InetAddress.getLoopbackAddress().getHostName(),
+        environment.getProperty("local.server.port"));
```

Comment:
@GalaHadi is it possible to use System.getProperty("server.port") here instead of wiring an @Environment?

Reply:
It returns null.

Reply:
Hmmm, I vaguely remember something being weird about this. If it returns null, doesn't fuse throw and error on startup?

Reply:
Nope, just prints the null out haha. 

`Started successfully, you can view swagger UI here: localhost:null/`

Reply:
I don't think this is the expected behavior, at least from the lines up above the ones you added. Can you use `args.containsOption("server.port")`? I think we want to make sure we're only wiring the Environment if completely necessary

Reply:
I think this is an order of operations thing -- I've seen System.getProperty("server.port") work elsewhere when called _after_ run() rather than inside it.

Reply:
@GalaHadi when you get a chance could you take a look at Constance's recommendation?

Reply:
Raj suggests running it in a debugger and seeing what that property is set to. If you want, I can stop by and we can take a look together.

Reply:
@GalaHadi have you had a chance to look at this?